mlflow/utils/__init__.py

```diff
@@ -268,3 +268,24 @@ def get_results_from_paginated_fn(paginated_fn, max_results_per_page, max_result
         else:
             break
     return all_results
+
+
+class AttrDict(dict):
```

Comment:
Perhaps we could subclass the deployment implementation from this generic abstract to get the specific Databricks naming convention? This could definitely be used elsewhere!

Reply:
updated!